apps/change-log.md

```diff
@@ -14,16 +14,28 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2023-05-11
+
+### Changes with software release 7.83.0
+
+#### <i class="fas fa-plus"></i> New
+
+* [`POST`/{shopId}/products/{productId}/variations](https://developer.epages.com/apps/api-reference/post-shopid-products-productid-variations.html)
+
+* Added data types `variation (create request)`, `variationAttribute (create request)` and `variationValue (create request)`.
```

Comment:
Are these for both NOW and Base? In some older changelog entries it says "for ePages Now only", so just double-checking here.

Reply:
@ywolpers I checked with the developers and it seems these are also "ePages Now only". I added this information and pushed the changes.